tests/test_models.py

```diff
@@ -7,29 +7,14 @@
 
 class TestAgent:
 
-    def test_unicode(self):
-        agent = factories.AgentFactory.build()
```

Comment:
Since we added the `__unicode__` methods back as `__str__`, it might make sense to add back these tests to test `str(agent)` etc.